packages/front-end/components/Experiment/BreakDownResults.tsx

```diff
@@ -94,19 +94,19 @@ const BreakDownResults: FC<{
         return {
           metric: newMetric,
           isGuardrail: !metrics.includes(metricId),
-          rows: results.map((d) => {
-            return {
+          rows: results
+            .map((d) => ({
               label: d.name,
               metric: newMetric,
               variations: d.variations.map((variation) => {
                 return variation.metrics[metricId];
               }),
               regressionAdjustmentStatus,
-            };
-          }),
+            }))
+            .filter((row) => row?.metric) as ExperimentTableRow[],
```

Comment:
Wouldn't it make more sense to filter before mapping so we aren't iterating over objects unnecessarily?

Reply:
hm we don't need .filter() at all here, removing...